web-common/src/features/connectors/olap/olap-config.ts

```diff
@@ -53,6 +53,12 @@ export function makeSufficientlyQualifiedTableName(
     case "pinot":
       // TODO
       return table;
+    case "mysql":
+      // MySQL uses database.table format (no schema concept like PostgreSQL)
+      if (database && database !== "default") {
+        return `${database}.${table}`;
+      }
+      return table;
```

Comment:
Given this code now applies to non-OLAP connectors, it seems this file should be renamed and moved out of the `connectors/olap` directory.